rfc-template.md

```diff
@@ -1,50 +1,58 @@
-{{
+{
 - Use this template when submitting a request for comment (RFC) proposal to the Good Docs Project.
 - See the [README](README.md) for more information about the process.
-}}
+}
 
 # Title of your proposal
 
 ## Proposed by
 
-{{Put your name and/or your Slack handle here.}}
+{Put your name and/or your Slack handle here.}
 
 ## Current status
 
-- [x] Draft {{Leave in a draft state until you submit your pull request.}}
-- [ ] Under discussion (until YYYY-MM-DD) {{Add date after selecting this status.}}
-- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
-- [ ] Accepted
-- [ ] Rejected
-- [ ] Implemented
-- [ ] Deferred
-- [ ] Withdrawn
+See the status indicated on the pull request description.
 
 
 ## Proposal overview
 
-{{Summarize your proposal in this section in about 1-2 paragraphs.}}
+{Summarize your proposal in this section in about 1-2 paragraphs.}

```

Comment:
💭 __thought/non-blocking:__ I'm assuming we are wrapping these in `{ ... }` braces thinking one day we will programmatically replace this?  I'd propose in the interim we use `>` which decorates these to use `<Blockquote />` so they end up looking like just callout/sub-headers.

example:

## Proposal Overview
> Summarize your proposal in this section in about _1-2 paragraphs_.

Reply:
Contributors can either just delete them, or keep them as guides/context info.